pkg/build/nodeimage/const_cni.go

```diff
@@ -20,7 +20,7 @@ package nodeimage
 The default CNI manifest and images are our own tiny kindnet
 */
 
-const kindnetdImage = "docker.io/kindest/kindnetd:v20221004-44d545d1"
+const kindnetdImage = "ghcr.io/verrazzano/kind-kindnetd:20230216193743-4b302893"
```

Comment:
Here and a few other spots, we should look at getting this passed in rather than baked in as a constant. Can do as a follow-up, but thinking we will be rebuilding things and we should be able to avoid changing the sources to pick up new ones.

Maybe do similar approach as what we do for the CLI version to get it supplied in by -X LDFLAGS

Reply:
Yea, that would be ideal. We can add that as a follow up if needed.